src/main/java/Duke.java

```diff
@@ -1,10 +1,265 @@
+                    continue;
+                }
+                String taskDescription;
+                try {
+                    taskDescription = extractDescription(userInput);
+                } catch (DukeException e) {
+                    System.out.println("____________________________________________________________");
+                    System.out.println("☹ OOPS!!! The description of a " + taskType + " cannot be empty.");
+                    System.out.println("____________________________________________________________");
+                    continue;
+                }
+                switch (taskType) {
+                case "todo":
+                    addTask(new Todo(taskDescription));
+                    break;
+                case "deadline":
+                    String date;
+                    try {
+                        date = extractDeadline(userInput);
+                    } catch (DukeException e) {
+                        System.out.println("____________________________________________________________");
+                        System.out.println("☹ OOPS!!! Please enter a deadline for your task!");
+                        System.out.println("____________________________________________________________");
+                        continue;
+                    }
+                    addTask(new Deadline(taskDescription, date));
+                    break;
+                case "event":
+                    String duration;
+                    try {
+                        duration = extractDuration(userInput);
+                    } catch (DukeException e) {
+                        System.out.println("____________________________________________________________");
+                        System.out.println("☹ OOPS!!! Please enter a duration for your task!");
+                        System.out.println("____________________________________________________________");
+                        continue;
+                    }
+                    addTask(new Event(taskDescription, duration));
+                    break;
+                }
+            }
+        } while(!isLastInput);
+        exit();
     }
+
+    private static String extractDescription(String userInput) throws DukeException {
+        int startIndex = userInput.indexOf(" ");
+        if (startIndex == -1) {
+            throw new DukeException();
+        }
+        int endIndex;
+        if (userInput.contains("/")) {
+            endIndex = userInput.indexOf("/");
+        } else {
+            endIndex = userInput.length();
+        }
+        String description = userInput.substring(startIndex+1, endIndex).trim();
+        if (description.equals("")) {
+            throw new DukeException();
+        }
+        return description;
+    }
+
+    private static boolean checkTaskListSize() {
+        if (taskCount == 0) {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
Consider renaming the boolean method to start with the prefix "is". Perhaps "isEmpty" or something similar to that would be more suitable?